src/zope/interface/tests/test_verify.py

```diff
@@ -557,5 +557,26 @@ class IDummyModule(Interface):
         self.assertRaises(DoesNotImplement,
                           self._callFUT, IDummyModule, dummy)
 
+    def test_staticmethod_hit_on_class(self):
+        from zope.interface import Interface
+        from zope.interface import provider
+        from zope.interface.verify import verifyObject
+
+        class IFoo(Interface):
+
+            def bar(a, b):
+                pass
```

Comment:
I think to make coverage happy you can change this to `raise NotImplementedError()`

Reply:
Whoops, usually I'm pretty good about that kind of thing. Thanks!